### PR TITLE
Pull Request directed at getting the Final State Processor and the Newest Updates to the SvtRawDataAnaProcessor into hpstr.

### PR DESCRIPTION
**Pull request by rodwyer100**
Recently the Final State Particle Processor was created in hpstr to allow us to associate tracks and Ecals Clusters with the hits that compose them. This idea had been thought of before, but the appropriate linkages that allowed one to refer to hits from a track had never been properly performed. This has been done here and been validated in a number of ways. Furthermore it has been used in conjunction with the RawSvtDataAnaProcessor to construct a couple plots and see how the pileUp pulse decision may be improved w.r.t hits we actually intend on included in analysis. All of these changes will be included in this PR.

### CODE REVIEW

File: processors/config/baseConfig.py [hunk 1]
```diff
@@ -23,7 +23,6 @@
 parser.add_argument('--infile', '-i', type=str, dest="inFilename", metavar='infiles', nargs="+",
                     help="Input files, specify on or more.")
```
Comment (cbravo135) [inline processors/config/baseConfig.py]: lets not change this file in this PR.

File: processors/config/kalSimpTuple_cfg.py [hunk 2]
```diff
@@ -187,6 +186,7 @@
 # If MC, get MCParticles
 if (not options.isData):
```
Comment (cbravo135) [inline processors/config/kalSimpTuple_cfg.py]: lets not change this file in this PR

File: processors/src/SvtRawDataAnaProcessor.cxx [hunk 3]
```diff
@@ -349,6 +408,7 @@ bool SvtRawDataAnaProcessor::process(IEvent* ievent) {
         }
         //std::cout<<count1<<std::endl;
         //std::cout<<count2<<std::endl;
+        std::cout<<"I got here 10"<<std::endl;
```
Comment (cbravo135) [inline processors/src/SvtRawDataAnaProcessor.cxx]: go through and take out all these random couts you have for debugging, or at least add an if(debug) to them.

File: processors/config/baseConfig.py (hunk 1 above)
Comment (rodwyer100) [reply processors/config/baseConfig.py]: Fixed.

File: processors/config/kalSimpTuple_cfg.py (hunk 2 above)
Comment (rodwyer100) [reply processors/config/kalSimpTuple_cfg.py]: Fixed

Comment (cbravo135) [reply processors/config/kalSimpTuple_cfg.py]: this file is still changed in this PR

File: processors/src/SvtRawDataAnaProcessor.cxx (hunk 3 above)
Comment (rodwyer100) [reply processors/src/SvtRawDataAnaProcessor.cxx]: I have taken out any that weren't commented and didn't mean something.

File: processors/config/fspTuple_cfg.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
+import HpstrConf
```
Comment (cbravo135) [inline processors/config/fspTuple_cfg.py:1]: So my thinking was more this cfg would just do FSPs (many analyses can be done on just fsp), and we add FSPs to recoTuple_cfg.py

File: processors/config/recoTuple_cfg.py [hunk 5]
```diff
@@ -138,6 +138,15 @@
 mcpart.parameters["mcPartCollLcio"] = 'MCParticle'
 mcpart.parameters["mcPartCollRoot"] = 'MCParticle'
 
+#FinalStateParticleProcessor
```
Comment (cbravo135) [inline processors/config/recoTuple_cfg.py:143]: What I want to do with this file is add FSPs, and also add a new option that will add hits or not depending on the option given. We can discuss in person how we want to deal with having the hits for tracks vs fsp, or both.

Comment (rodwyer100) [reply processors/config/recoTuple_cfg.py:143]: Done

**Review by cbravo135 — APPROVED** [review]

**Pull request merged**